pyproject.toml

```diff
@@ -26,6 +26,7 @@ dependencies = [
   "beautifulsoup4",
   "createrepo_c",
   "gitpython",
+  "jaraco.text",
```

Comment:
With the helper functions potentially gone this import likely won't be needed and we also shouldn't depend on a library that pkg_resources's maintainers/creators created just to be able to keep vendoring it within setuptools.

Reply:
I advised ianrochapg to directly use 'jaraco.text' - I didn't realize that it was so closely related to 'pkg_resources', and both of us were quite puzzled as to why it was being vendored there in the first place :D